frontend/packages/core/src/Resolver/hydrator.tsx

```diff
@@ -108,6 +108,11 @@ const OptionField = (
     });
   }, []);
 
+  const hasError = field.metadata.required && options.length === 0;
+  if (hasError) {
+    options = ["Error: Missing options for required field"];
```

Comment:
Is there a breadcrumb that could be left here to describe what the missing option/options might be?

Reply:
interesting idea. that information would need to come from server side.

would need to add some kind of error state to the schema for rendering. i'll think on it and file a followup issue.